[2015-11-23 05:28:21] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2776): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2053): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2036): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\hrm\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-11-23 05:28:21] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2060): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-11-23 05:28:24] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2776): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2053): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2036): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\hrm\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-11-23 05:28:24] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2060): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-11-23 05:29:00] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2776): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2053): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2036): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\hrm\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-11-23 05:29:00] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2060): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-11-23 05:31:22] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2776): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2053): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2036): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\hrm\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-11-23 05:31:22] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2060): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-11-23 05:32:30] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2776): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2053): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2036): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\hrm\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-11-23 05:32:30] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2060): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-11-23 05:32:31] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2776): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2053): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2036): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\hrm\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-11-23 05:32:31] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2060): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-11-23 05:34:05] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2776): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2053): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2036): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\hrm\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-11-23 05:34:05] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2060): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-11-23 05:34:13] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2776): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2053): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2036): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\hrm\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-11-23 05:34:13] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:6784
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1086): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1131): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1115): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1101): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1039): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(1577): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2060): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\hrm\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-11-23 05:41:42] production.ERROR: exception 'InvalidArgumentException' with message 'View [.pages.dashboard] not found.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:13482
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(13455): Illuminate\View\FileViewFinder->findInPaths('.pages.dashboar...', Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(13583): Illuminate\View\FileViewFinder->find('.pages.dashboar...')
#2 C:\xampp\htdocs\hrm\vendor\laravel\framework\src\Illuminate\Foundation\helpers.php(648): Illuminate\View\Factory->make('/pages/dashboar...', Array, Array)
#3 C:\xampp\htdocs\hrm\app\Http\routes.php(15): view('/pages/dashboar...')
#4 [internal function]: App\Providers\RouteServiceProvider->{closure}()
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7572): call_user_func_array(Object(Closure), Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7559): Illuminate\Routing\Route->runCallable(Object(Illuminate\Http\Request))
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7215): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9287): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7216): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#13 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7204): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7189): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#15 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2106): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9287): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2726): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#19 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#20 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#21 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(12579): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#22 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#24 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(11210): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#27 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(12316): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#30 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(12255): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#33 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2776): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#35 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#36 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#37 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#38 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2053): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#39 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2036): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#40 C:\xampp\htdocs\hrm\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#41 {main}  
[2015-11-23 05:48:55] production.ERROR: exception 'InvalidArgumentException' with message 'View [.pages.dashboard] not found.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:13482
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(13455): Illuminate\View\FileViewFinder->findInPaths('.pages.dashboar...', Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(13583): Illuminate\View\FileViewFinder->find('.pages.dashboar...')
#2 C:\xampp\htdocs\hrm\vendor\laravel\framework\src\Illuminate\Foundation\helpers.php(648): Illuminate\View\Factory->make('/pages/dashboar...', Array, Array)
#3 C:\xampp\htdocs\hrm\app\Http\routes.php(15): view('/pages/dashboar...')
#4 [internal function]: App\Providers\RouteServiceProvider->{closure}()
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7572): call_user_func_array(Object(Closure), Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7559): Illuminate\Routing\Route->runCallable(Object(Illuminate\Http\Request))
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7215): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9287): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7216): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#13 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7204): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7189): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#15 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2106): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9287): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2726): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#19 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#20 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#21 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(12579): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#22 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#24 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(11210): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#27 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(12316): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#30 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(12255): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#33 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2776): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#35 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#36 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#37 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#38 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2053): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#39 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2036): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#40 C:\xampp\htdocs\hrm\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#41 {main}  
[2015-11-23 05:48:57] production.ERROR: exception 'InvalidArgumentException' with message 'View [.pages.dashboard] not found.' in C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php:13482
Stack trace:
#0 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(13455): Illuminate\View\FileViewFinder->findInPaths('.pages.dashboar...', Array)
#1 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(13583): Illuminate\View\FileViewFinder->find('.pages.dashboar...')
#2 C:\xampp\htdocs\hrm\vendor\laravel\framework\src\Illuminate\Foundation\helpers.php(648): Illuminate\View\Factory->make('/pages/dashboar...', Array, Array)
#3 C:\xampp\htdocs\hrm\app\Http\routes.php(15): view('/pages/dashboar...')
#4 [internal function]: App\Providers\RouteServiceProvider->{closure}()
#5 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7572): call_user_func_array(Object(Closure), Array)
#6 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7559): Illuminate\Routing\Route->runCallable(Object(Illuminate\Http\Request))
#7 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7215): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9287): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7216): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#13 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7204): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(7189): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#15 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2106): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9287): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2726): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#19 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#20 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#21 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(12579): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#22 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#24 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(11210): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#27 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(12316): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#30 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(12255): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#33 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2776): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#35 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9279): call_user_func_array(Array, Array)
#36 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#37 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(9269): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#38 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2053): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#39 C:\xampp\htdocs\hrm\bootstrap\cache\compiled.php(2036): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#40 C:\xampp\htdocs\hrm\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#41 {main}  
[2015-11-23 05:51:23] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '<<' (T_SL)' in C:\xampp\htdocs\hrm\app\Http\routes.php:16
Stack trace:
#0 {main}  
[2015-11-23 06:25:56] production.ERROR: exception 'InvalidArgumentException' with message 'Command "migrate:make" is not defined.

Did you mean one of these?
    migrate:reset
    migrate:refresh
    migrate:status
    migrate:rollback
    migrate:install
    migrate' in C:\xampp\htdocs\hrm\vendor\symfony\console\Application.php:549
Stack trace:
#0 C:\xampp\htdocs\hrm\vendor\symfony\console\Application.php(192): Symfony\Component\Console\Application->find('migrate:make')
#1 C:\xampp\htdocs\hrm\vendor\symfony\console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 C:\xampp\htdocs\hrm\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(100): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 C:\xampp\htdocs\hrm\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
